test/parallel/test-stream-base-prototype-accessors.js

```diff
@@ -10,18 +10,33 @@ const assert = require('assert');
 // Or anything that calls StreamBase::AddMethods when setting up its prototype
 const TTY = process.binding('tty_wrap').TTY;
 
-// Should throw instead of raise assertions
 {
-  const msg = /TypeError: Method \w+ called on incompatible receiver/;
+  // Should throw instead of raise assertions
   assert.throws(() => {
     TTY.prototype.bytesRead;
-  }, msg);
+  }, TypeError);
 
   assert.throws(() => {
     TTY.prototype.fd;
-  }, msg);
+  }, TypeError);
 
   assert.throws(() => {
     TTY.prototype._externalStream;
-  }, msg);
+  }, TypeError);
+
+  // Should not throw for Object.getOwnPropertyDescriptor
+  assert.strictEqual(
+    typeof Object.getOwnPropertyDescriptor(TTY.prototype, 'bytesRead'),
+    'object'
+  );
+
+  assert.strictEqual(
+    typeof Object.getOwnPropertyDescriptor(TTY.prototype, 'fd'),
+    'object'
+  );
+
+  assert.strictEqual(
+    typeof Object.getOwnPropertyDescriptor(TTY.prototype, '_externalStream'),
+    'object'
+  );
```

Comment:
Similar tests should exist for `Object.getOwnPropertyDescriptor(crypto.SecureContext.prototype, '_external')` and `Object.getOwnPropertyDescriptor(crypto.Connection.prototype, '_external')` where `crypto = process.binding('crypto')`.

Reply:
:+1: Added.